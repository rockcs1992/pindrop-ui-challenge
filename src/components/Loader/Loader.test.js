import React from 'react';
import { render } from 'enzyme';
import Loader from './index';

describe('Loader', () => {
    it('should render correctly', () => {
        const tree = render(<Loader />);
        expect(tree).toMatchSnapshot();
    })
})