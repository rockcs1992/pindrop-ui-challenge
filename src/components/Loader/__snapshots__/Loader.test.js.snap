// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<div
  class="sc-bdVaJa khUSez"
>
  <div
    class="sc-bwzfXH kxxnux"
  />
</div>
`;
